pages/SignIn/index.tsx

```diff
@@ -7,85 +7,108 @@ import google from "@/public/google.png";
 import kakao from "@/public/kakaoSocial.png";
 
 import { ChangeEvent, FormEvent, useState } from "react";
-import { useRouter } from "next/router";
+import { useRouter } from "next/navigation";
 
 import styles from "@/pages/SignIn/signIn.module.css";
 import Image from "next/image";
 import Link from "next/link";
-import emailCheck from "@/lib/emailCheck";
-import axios from "@/lib/axios";
+import emailCheck from "@/utils/isEmail";
+import axios from "@/api/axios";
 import AccessTokenChecker from "@/components/AccessTokenChecker/AccessTokenChecker";
 
 function SignIn() {
-  const [emailInputValue, setEmailInputValue] = useState("");
-  const [passwordInputValue, setPasswordInputValue] = useState("");
-  const [emailError, setEmailError] = useState("");
-  const [passwordError, setPasswordError] = useState("");
+  const [inputValues, setInputValues] = useState({
+    email: "",
+    password: "",
+  });
+
+  const [errors, setErrors] = useState({
+    emailError: "",
+    passwordError: "",
+  });
 
   const router = useRouter();
 
   const handleEmailInputValueChange = (e: ChangeEvent<HTMLInputElement>) => {
-    setEmailInputValue(e.currentTarget.value);
+    setInputValues(prev => ({
+      ...prev,
+      email: e.target.value,
+    }));
   };
 
   const handlePasswordInputValueChange = (e: ChangeEvent<HTMLInputElement>) => {
-    setPasswordInputValue(e.currentTarget.value);
+    setInputValues(prev => ({ ...prev, password: e.target.value }));
   };
 
   const handleEmailBlur = (e: ChangeEvent<HTMLInputElement>) => {
     if (!e.target.value) {
-      setEmailError("이메일을 입력해 주세요.");
-    } else if (!emailCheck(e.target.value)) {
-      setEmailError("올바른 이메일 주소가 아닙니다.");
-    } else {
-      setEmailError("");
+      setErrors(prev => ({
+        ...prev,
+        emailError: "이메일을 입력해 주세요.",
+      }));
+      return;
+    }
+    if (!emailCheck(e.target.value)) {
```

Comment:
조건문이 나열될때는 개행을 넣어주시면 좋습니다~
```suggestion
      setErrors(prev => ({
        ...prev,
        emailError: "이메일을 입력해 주세요.",
      }));
      return;
    }
    
    if (!emailCheck(e.target.value)) {
```